eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -86,3 +86,21 @@ steps:
       -GitHubUsers "$(${{ parameters.GHReviewersVariable }})"
       -GitHubTeams "$(${{ parameters.GHTeamReviewersVariable }})"
       -PRNumber "$(Submitted.PullRequest.Number)"
+
+- task: PowerShell@2
+  displayName: Add Assignee to Pull Request
+  condition: and(succeeded(), eq(variables['HasChanges'], 'true'))
+  continueOnError: true
+  inputs:
+    pwsh: true
+    workingDirectory: ${{ parameters.WorkingDirectory }}
+    filePath: ${{ parameters.ScriptDirectory }}/add-issue-assignees.ps1
+    arguments: >
+      -RepoOwner "${{ parameters.RepoOwner }}"
+      -RepoName "${{ parameters.RepoName }}"
+      -AuthToken "$(azuresdk-github-pat)"
+      -GitHubUsers "$(${{ parameters.GHReviewersVariable }})"
```

Comment:
I'm starting to wonder if we shouldn't just add this as a parameter to the Submit-PullRequest.ps1 like we did with the labels. We should consider the same thing for the reviewers.

Reply:
It will require updating how the code is consumed else where.